content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -54,8 +54,8 @@ to use this feature with Kubernetes stateless pods:
 
 * CRI-O: version 1.25 (and later) supports user namespaces for containers.
 
-Please note that containerd v1.7 supports user namespaces for containers,
-compatible with Kubernetes {{< skew currentPatchVersion >}}. It should not be used
+Please note that containerd v1.7 (and later) supports user namespaces for containers,
+Kubernetes {{< skew currentPatchVersion >}} introduced support for user namespaces in container runtimes. It should be used
 with Kubernetes 1.27 (and later).
```

Comment:
Sorry, but this looks like the wrong fix. Check with SIG Node to find out what advice we want to give to readers.

Reply:
Anyone is welcome to make the fix. You need to make sure that the advice is technically accurate; if you're not sure what the true compatibility situation is, you might want to either:
- step back from working on this PR (which allows someone who does know the details to join in instead)
- ask the relevant SIG for advice

For this page, the relevant SIG is [SIG Node](https://github.com/kubernetes/community/blob/master/sig-node/README.md).